src/GitPrompt.ps1

```diff
@@ -101,6 +101,8 @@ $global:GitPromptSettings = [pscustomobject]@{
     DefaultPromptDebugSuffix                    = ' [DBG]$(''>'' * ($nestedPromptLevel + 1)) '
     DefaultPromptEnableTiming                   = $false
 
+    AbbreviateHomeDirectory                     = $true

```

Comment:
Given the option, I personally prefer 'Directory', so unless you feel strongly about using `Folder` to save the three characters, I'd prefer to use 'Directory'.

I'll change to `DefaultPromptAbbreviateHomeDirectory`, and you can veto as necessary.